warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -373,7 +366,6 @@ public class QueryOptions implements OptionDescriber {
     protected boolean compressResults = false;
     
     protected Boolean compressedMappings = false;
-    protected boolean limitOverride = false;
```

Comment:
It appears that this variable isn't used anywhere and that's why you removed it.  Just want to make sure I understand this correctly.

Reply:
Correct, this is just removing an unused variable.